### ISSUE TITLE
Use As Default reCAPTCHA

### ISSUE THREAD

**Issue by ghost** [moxi9/reCAPTCHA#2]
Is it possible to have this app generate in all the locations a CAPTCHA is on our sites?

For example the: "Forgot My Password" section. It uses the old version and a number of people have told me it gives 'incorrect'. Example shown below:

![image](https://cloud.githubusercontent.com/assets/6254165/8711994/f3bb2536-2b24-11e5-9637-376094aa9fe5.png)

![image](https://cloud.githubusercontent.com/assets/6254165/8711995/f641091a-2b24-11e5-88bd-ba7989df6399.png)

**Comment by RaymondBenc** [comment]
Yup, the goal is for the app to replace the default captcha. We just wanted to get the app out there so we least cover the registration page. I'll update you once we get this added.

**Comment by ghost** [comment]
Great! Thanks. :)
